classes/Conf/class.xoctConfFormGUI.php

```diff
@@ -159,6 +166,11 @@ public static function checkItem($item)
      */
     protected function initAPISection()
     {
+        global $DIC;
+        $main_tpl = $DIC->ui()->mainTemplate();
+        $main_tpl->addCss($this->plugin->getStyleSheetLocation('default/password_toggle.css'));
+        $main_tpl->addJavaScript($this->plugin->getStyleSheetLocation('default/password_toggle.min.js'));
```

Comment:
this should be changes to ->addOnloadCode() and you should pass something like an `init('#elemt_id')` method. please have a look to my comment in password_toggle.js